examples/applications/restapi/txstatewebsocket.py

```diff
@@ -0,0 +1,117 @@
+# Adapted from https://stackoverflow.com/questions/48695294/how-can-i-detect-the-closure-of-an-python-aiohttp-web-socket-on-the-server-when
+# while looking at ways to handle the issue of when the client exits uncleanly
+import json
+import uuid
+import aiohttp
+from aiohttp import web
+from bitcoinx import hex_str_to_hash
+
+from electrumsv.logs import logs
+from electrumsv.restapi import Fault
+from electrumsv.wallet import AbstractAccount
+
+from examples.applications.restapi.handler_utils import VNAME
+from examples.applications.restapi.errors import Errors
+
+
+class WSClient(object):
+
+    def __init__(self, ws_id: str, websocket: web.WebSocketResponse, account: AbstractAccount):
+        self.ws_id = ws_id
+        self.websocket = websocket
+        self.account = account
```

Comment:
Additionally, you should document the lifecycle of the account reference if this is the case. Leaking wallets is something we want to avoid and I have put a lot of work into tracking reference leaks.

Reply:
I will leave this comment in the code:

```
        # If 'account' is garbage collected prior to fully stopping the ElectrumSV application (
        # and RESTAPI), will need to force close websocket connections associated with that wallet.
        # There is currently no RESTAPI endpoint that allows stopping a wallet but would need to
        # be responsible for this cleanup process (otherwise would get a ReferenceError when
        # fetching transactions)
```